packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -76,11 +76,11 @@ export class TileDoctype extends Doctype {
             throw new Error('No DID authenticated')
         }
 
-        const supported_chains = await context.api.getSupportedChains()
-        if ('chainId' in metadata && !supported_chains.includes(metadata.chainId)) {
-            throw new Error("Requested chainId '" + metadata.chainId + "' is not supported. Supported chains are: '" + supported_chains.join("', '") + "'")
+        const supportedChains = await context.api.getSupportedChains()
+        if ('chainId' in metadata && !supportedChains.includes(metadata.chainId)) {
+            throw new Error("Requested chainId '" + metadata.chainId + "' is not supported. Supported chains are: '" + supportedChains.join("', '") + "'")
         }
-        metadata.chainId = metadata.chainId ?? supported_chains[0]
+        metadata.chainId = metadata.chainId ?? supportedChains[0]
```

Comment:
Can we have a hardcoded chainId default (`eip155:3` for now) that you can change with a config? Right now it's unclear if this will be inmemory or ethereum if an ethereum service is configured.

Reply:
yep 👍

Reply:
This still grabs the chainId from an array which we don't neccesarily control the order over. This can cause unknown errors. Also we likely need to do something similar for the `caip10-link` doctype.

Reply:
We are ordering the list on the `Ceramic` side. And yes, this should be possibly done in `caip10-link` doctype.

What was your suggestion about hardcoding the `eip155:3` in these places?